src/pretalx/submission/models/submission.py

```diff
@@ -1041,6 +1041,11 @@ def private_resources(self):
     def public_resources(self):
         return self.active_resources.filter(is_public=True)
 
+    @cached_property
+    def available_public_resources(self):
```

Comment:
I don't think we want a new property here, and instead use this logic in `public resources`. This makes sure that any plugin relying on `public_resources` will continue to only receive the correctly filtered list.